imagetest/fixtures.go

```diff
@@ -220,6 +221,12 @@ func (t *TestVM) EnableSecureBoot() {
 // subnetwork. If subnetwork is empty, not using subnetwork, in this case
 // network has to be in auto mode VPC.
 func (t *TestVM) SetCustomNetwork(network *Network, subnetwork *Subnetwork) error {
+	return t.SetCustomNetworkWithNetworkIP(network, subnetwork, "")
+}
+
+// SetCustomNetworkWithNetworkIP set current test VMs in workflow using provided
+// network, subnetwork and a static IP selected by user.
+func (t *TestVM) SetCustomNetworkWithNetworkIP(network *Network, subnetwork *Subnetwork, networkIP string) error {
```

Comment:
How about accepting `compute.NetworkInterface` instead of the individual fields? That allows you to avoid the somewhat-confusing (to me anyway :)  )  usage of empty strings and nils, such as `SetCustomNetworkWithNetworkIP(network1, nil, "")`

Reply:
We don't want to explore the whole GCE struct to end user to setup for test. However we could add a wrapper Network Interface struct as we did for Network and Subnetwork.

Reply:
This wrapper method has be removed as we set network Ip latter instead set here. This method is just for create nic (keep original functional)

Reply:
lgtm (I don't have permission to resolve the comment thread)